src/ui/default-prop-types.js

```diff
@@ -281,10 +281,23 @@ export const documentInputSectionCommentShape = PropTypes.shape({
 
 export const sectionInputShape = PropTypes.shape({
   data: PropTypes.object,
-  comments: PropTypes.arrayOf(documentInputSectionCommentShape)
+  files: PropTypes.arrayOf(PropTypes.shape({
+    key: PropTypes.string.isRequired,
+    name: PropTypes.string.isRequired,
+    size: PropTypes.number.isRequired,
+    type: PropTypes.string.isRequired,
+    url: PropTypes.string.isRequired
+  })).isRequired,
+  comments: PropTypes.arrayOf(PropTypes.shape({
```

Comment:
Here the setup from before needs to stay (`comments: PropTypes.arrayOf(documentInputSectionCommentShape` ), I imagine there was a merge conflict.

Reply:
Actually I inlined it, but reverted now and also extracted the shape for the files array.